.buildkite/common.py

```diff
@@ -324,10 +329,12 @@ def to_json(self):
         """Serialize the pipeline to JSON"""
         return json.dumps(self.to_dict(), indent=4, sort_keys=True, ensure_ascii=False)
 
-    def devtool_test(self, devtool_opts=None, pytest_opts=None):
+    def devtool_test(self, devtool_opts=None, pytest_opts=None, devtool_no_build=True):
         """Generate a `devtool test` command"""
         cmds = []
-        parts = ["./tools/devtool -y test", "--no-build"]
+        parts = ["./tools/devtool -y test"]
+        if devtool_no_build:
```

Comment:
Do we really need this parameter? Couldn't this check be `if self.shared_build is not None`?

Reply:
you are right. I switched to your suggestion.